cloud-optimized-geotiffs/writing-cogs-in-python.ipynb

```diff
@@ -0,0 +1,452 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.            print(mem)
</pre><p>Code needs more comments explaining what is going on.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Thank you @wildintellect for the review comments. I've addressed this in my recent commit. Requesting re-review.